torch/fx/graph.py

```diff
@@ -926,7 +926,8 @@ def emit_node(node : Node):
                 if global_name == 'getattr' and \
                    isinstance(node.args, tuple) and \
                    isinstance(node.args[1], str) and \
-                   node.args[1].isidentifier():
+                   node.args[1].isidentifier() and \
+                   len(node.args) == 2:
                     # pretty print attribute access
                     body.append(f'{repr(node)}{maybe_type_annotation} = {_format_target(repr(node.args[0]), node.args[1])}')
```

Comment:
So I think this is baking in the existence of the attribute at trace time rather than resolving it at runtime. Can you print out the `code` in the test case to confirm?

What I was thinking of was actually checking in the branches here if `len(args) == 2` and emitting an explicit `getattr` node in the generated code. Then, the existence of the attribute would be checked at runtime

Reply:
This is the generated `code` in the test case:
`
def forward(self, x):
    _tensor_constant0 = self._tensor_constant0
    getattr_1 = getattr(x, 'nonexistent_attr', _tensor_constant0);  x = _tensor_constant0 = None
    return getattr_1
`
I think the first line after the `if` block,
`body.append(f'{repr(node)}{maybe_type_annotation} = {global_name}({_format_args(node.args, node.kwargs)})')`
generated the `getattr` call.  It's a good idea to check the existence of the attribute at runtime; I'll make that change.  Thanks.

Reply:
Sorry the previous formatting was bad.  This is the output of `print(traced.code)`:

`def forward(self, x):`
`    _tensor_constant0 = self._tensor_constant0`
`    getattr_1 = getattr(x, 'nonexistent_attr', _tensor_constant0);  x = _tensor_constant0 = None`
`    return getattr_1`